admin-base/ui.apps/src/main/content/jcr_root/apps/field/collection/template.vue

```diff
@@ -23,216 +23,257 @@
   #L%
   -->
 <template>
-	<div class="wrap">
-    <label>{{schema.title}} </label>
-    <ul v-if="!schema.preview" class="collapsible" v-bind:class="schema.multifield ? 'multifield' : 'singlefield'" ref="collapsible">
-        <li v-for="(item, index) in value" v-bind:class="getItemClass(item, index)" v-bind:key="item.path"> {{item._opDelete}}
-            <div
-              ref="header"
-              class="collapsible-header"
-              draggable="true"
-              v-on:dragstart = "onDragStart(item, index, $event)"
-              v-on:dragover.prevent  ="onDragOver"
-              v-on:dragenter.prevent ="onDragEnter"
-              v-on:dragleave.prevent ="onDragLeave"
-              v-on:drop.prevent      ="onDrop(item, index, $event)"
-              v-on:click.stop.prevent="onSetActiveItem(index)">
-                <i class="material-icons">drag_handle</i>
-                <span v-if="schema.multifield">{{itemName(item, index)}}</span>
-                <input
-                  v-else
-                  ref="input"
-                  v-model="value[index]">
-                <i class="material-icons delete-icon" v-on:click="onRemoveItem(item, index)">delete</i>
-            </div>
-            <transition
-              v-on:enter="enter"
-              v-on:leave="leave"
-              v-bind:css="false">
-              <div v-if="schema.multifield && activeItem === index" class="collapsible-body">
-                  <vue-form-generator
-                    :schema="schema"
-                    :model="prepModel(item, schema)"></vue-form-generator>
-              </div>
-            </transition>
-        </li>
-        <button type="button" class="btn-flat btn-add-item" v-on:click="onAddItem">
-          <i class="material-icons">add</i>
-        </button>
+  <div class="wrap">
+    <label>{{ schema.title }} </label>
+    <ul v-if="!schema.preview"
+        class="collapsible"
+        v-bind:class="schema.multifield ? 'multifield' : 'singlefield'"
+        ref="collapsible">
+      <li v-for="(item, index) in value"
+          class="collection-item"
+          v-bind:class="getItemClass(item, index)"
+          v-bind:key="`collection-item-${index}`"
```

Comment:
fixed the key here

Reply:
unsing an index is technically bad and can lead to update issues when reordering - andy chance we have something unique in tht item we can use?